src/test/java/com/wikia/webdriver/testcases/discussions/ReportingPostTests.java

```diff
@@ -81,54 +63,78 @@ public void anonUserOnMobileCanNotReportPostOnUserPostsPage() {
+    createAndReportPostRemotelyAsFristUser();
 
+    PostsListPage page = new PostsListPage().open();
     Assertion.assertFalse(postHasReportedIndicator(page), NO_REPORTED_INDICATOR_ON_POST_MESSAGE);
   }
 
-  @Test(groups = "discussions-anonUserMobileReporting",
-      dependsOnMethods = "userOnMobileCanReportPostOnPostDetailsPage")
+  @Test(groups = "discussions-anonUserMobileReporting")
   @Execute(asUser = User.ANONYMOUS)
   @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
   public void anonUserOnMobileCanNotSeeReportedPostOnPostDetailsPage() {
-    PostDetailsPage page = new PostDetailsPage().open(testsPostData.get(POST_ON_MOBILE_ON_POST_DETAILS).getId());
+    PostEntity.Data data = createAndReportPostRemotelyAsFristUser();
 
+    PostDetailsPage page = new PostDetailsPage().open(data.getId());
     Assertion.assertFalse(postHasReportedIndicator(page), NO_REPORTED_INDICATOR_ON_POST_MESSAGE);
   }
 
-  @Test(groups = "discussions-anonUserMobileReporting",
-      dependsOnMethods = "userOnMobileCanReportPostOnUserPostsPage")
+  @Test(groups = "discussions-anonUserMobileReporting")
   @Execute(asUser = User.ANONYMOUS)
   @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
   public void anonUserOnMobileCanNotSeeReportedPostOnUserPostsPage() {
-    UserPostsPage page = new UserPostsPage().open(testsPostData.get(POST_ON_MOBILE_ON_USER_PAGE).getAuthorId());
+    PostEntity.Data data = createAndReportPostRemotelyAsFristUser();
 
+    UserPostsPage page = new UserPostsPage().open(data.getAuthorId());
     Assertion.assertFalse(postHasReportedIndicator(page), NO_REPORTED_INDICATOR_ON_POST_MESSAGE);
   }
 
-  @Test(groups = "discussions-anonUserMobileReporting",
-      dependsOnMethods = {
-          "userOnMobileCanReportPostOnPostDetailsPage",
-          "anonUserOnMobileCanNotSeeReportedPostOnPostDetailsPage",
-          "userOnMobileCannotSeeReportedIndicatorOnPostsReportedByAnotherUserOnPostDetailsPageAndCanReportThatPost",
-          "moderatorOnMobileCanApproveReportedPostOnPostDetailsPage",
-          "userOnMobileCannotReReportPostOnPostDetailsPage",
-          "userOnMobileCanReportApprovedPostOnPostDetailsPage",
-          "moderatorOnMobileCanDeleteReportedPostOnPostDetailsPage"})
+  @Test(groups = "discussions-anonUserMobileReporting")
+  @Execute(asUser = User.ANONYMOUS)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void anonUserOnMobileCanNotSeeDeletedPostOnPostsListPage() {
+    PostEntity.Data data = createAndReportPostRemotelyAsFristUser();
+    reportPostRemotelyAsSecondUser(data);
+    validatePostRemotelyAsDiscussionsModerator(data);
+    reportPostRemotelyAsThirdUser(data);
+    deletePostRemotelyAsDiscussionsModerator(data);
+
+    PostEntity post = new PostsListPage().open().getPost().findPostById(data.getId());
+    Assertion.assertNull(post, "Anonymous user should not see deleted post.");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Anonymous user should not see deleted post." 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)